EventsExpress.Core/NotificationHandlers/RegisterVerificationHandler.cs

```diff
@@ -58,11 +52,10 @@ public async Task Handle(RegisterVerificationMessage notification, CancellationT
 
                 var x = _cacheHepler.GetValue(notification.User.Id);
 
-                Debug.WriteLine(x.Token);
             }
-            catch(Exception ex)
+            catch
             {
-                Debug.WriteLine(ex.Message);
+               
```

Comment:
Empty catch statements are very bad. You should always do something, otherwise why catching exceptions? Please consider injecting `ILogger<T>` interface and logging the exception explicitly, so we could debug and fix it if this happens in production.

Reply:
We need to discuss it and create story for logging integration. I guess nothing related work done before

Reply:
Well, the thing is, that `ILogger<T>` is a default Microsoft interface that is implemented by most .Net core logging frameworks. By default, without any loggers, it outputs into std out. This is enough for this usecase. We can create the story further on.